### ISSUE TITLE
[BUG] pg_bsd_indentPath not working as expected

### ISSUE THREAD

**Issue by zhjwpku** [ashenBlade/postgres-dev-helper#2]
**Describe the bug**
I'm using meson out of tree build, so the pg_bsd_indent is not in postgres src.

The postgres src directory is: /home/postgres/postgres/
and the build directory is: /home/postgres/build
I set `"postgresql-hacker-helper.pg_bsd_indentPath": "/home/postgres/build/src/tools/pg_bsd_indent/pg_bsd_indent"`, but
it seems not work.

VSCode give out the following error: `Seems like pg_bsd_indent is not build yet. Formatting is not supported without it. Build?`,
I search the code a little bit, the error is from `findExistingPgbsdindent`, but before that is should call `Configuration.getCustomPgbsdindentPath()` in `getPgbsdindent`, which seems not working.

I'm skeptical about the following code, but I don't know how to debug vscode extension, it will be great if you(@ashenBlade) can check this, thanks in advance.

```
    static ConfigSections = {
        TopLevelSection: this.ExtensionName,
        NodeTagFiles: 'nodeTagFiles',
        LogLevel: 'logLevel',
        PgindentPath: 'pgindentPath',
        PgbsdindentPath: 'pgindentPath',
        SrcPath: 'srcPath'
    };
```

**Comment by ashenBlade** [comment]
Hi! Thanks for report. Seems that you are right about root cause of this bug. I will fix this ASAP

**Comment by ashenBlade** [comment]
Released v1.4.2 with this bug fix

**Comment by zhjwpku** [comment]
> Released v1.4.2 with this bug fix

It works now, thanks for the quick fix :)
